docs/react/faq.en-US.md

```diff
@@ -59,56 +61,56 @@ Try [Space](https://ant.design/components/space/) component to make them aligned
+### I set `dependencies.antd` as the git repository in `package.json`, but it doesn't work.
 
-Yes, please install `antd` with npm or yarn.
+Please install `antd` with either npm or yarn.
 
-### `message` and `notification` is lower case, but other components are capitalized. Typo?
+### `message` and `notification` is lower case, but other components are capitalized. Is this a typo?
 
-No, as `message` is just a function, not a React Component.
+No, `message` is just a function, not a React Component, thus it is not a typo that it is in lower case.
 
 ### `antd` doesn't work well in mobile.
 
-Please check [And Design Mobile](http://mobile.ant.design) for details. `antd` has not been optimized to do so. You may try [react-component](https://github.com/react-component/), those repositories which start with 'm-' 'rn-' are designed for mobile.
+Please check [And Design Mobile](http://mobile.ant.design) as a possible solution, as `antd` has not been optimized to work well on mobile. You can also try the [react-component](https://github.com/react-component/) repositories which start with 'm-' 'rn-', which are also designed for mobile.
 
 ### Does `antd` supply standalone files like 'React'?
 
-Yes, you can [import `antd` with script tag](https://ant.design/docs/react/introduce#Import-in-Browser). But we recommend using `npm` to import `antd`, it is simple and easy to maintain.
+Yes, you can [import `antd` with script tag](https://ant.design/docs/react/introduce#Import-in-Browser), but we recommend using `npm` to import `antd`, as it is simple and easy to maintain.
 
 ### I can't visit `icon` in my network environment.
 
 You should deploy the iconfont files to your network by following this [example](https://github.com/ant-design/antd-init/tree/7c1a33cadb98f2fd8688fe527dd7f98215b9bced/examples/local-iconfont). [#1070](https://github.com/ant-design/ant-design/issues/1070)
 
-After 3.9.x [we are using svg icon](/components/icon#svg-icons), so you don't need to deploy iconfont locally anymore.
+After 3.9.x [we will also switch to using svg icons](/components/icon#svg-icons), so you won't need to deploy iconfont locally anymore as well.
 
 ### How do I extend antd's components?
 
 If you need some features which should not be included in antd, try to extend antd's component with [HOC](https://gist.github.com/sebmarkbage/ef0bf1f338a7182b6775). [more](https://medium.com/@dan_abramov/mixins-are-dead-long-live-higher-order-components-94a0d2f9e750#.eeu8q01s1)
 
-### How do I fix dynamic style when open Content Security Policy (CSP)?
+### How do I fix dynamic styles while using a Content Security Policy (CSP)?
 
 You can configure `nonce` by [ConfigProvider](/components/config-provider/#Content-Security-Policy).
 
-### When I set `mode` to DatePicker/RangePicker, I cannot select year or month anymore?
+### When I set `mode` to DatePicker/RangePicker, why can I not select a year or month anymore?
```

Comment:
should it be `why can't I`?

Reply:
@07akioni In this case either would work -- in my mind `not` was connected more to `select` (e.g. `not select`) vs being connected to myself (`why can not I`).